src/servers/main.cc

```diff
@@ -247,31 +282,105 @@ StartHttpService(nvidia::inferenceserver::InferenceServer* server)
   return std::move(service);
 }
 
+std::unique_ptr<nvidia::inferenceserver::GRPCServer>
```

Comment:
You need to "auto format" to fix these long lines. See https://docs.nvidia.com/deeplearning/sdk/tensorrt-inference-server-master-branch-guide/docs/contribute.html#coding-convention